src/main/java/com/github/monster860/fastdmm/FastDMM.java

```diff
@@ -805,4 +870,59 @@ private void loop() {
 			Display.sync(60);
 		}
 	}
+
+	private String getAppData() {
```

Comment:
I guess that could work, although it's not really conventional. I'll just add a dot prefix to the foldername.